docker_setup_scripts/opensuse.sh

```diff
@@ -46,12 +51,15 @@ packages=(
   xz
   # groff provides the soelim program needed by openldap build.
   groff
+  bazel5.2
```

Comment:
We don't really build on OpenSUSE and we should probably get rid of this script for simplicity (we can build on CentOS 7 and install on any other Linux). You can skip adding Bazel to OpenSUSE.